cmd/docker/main.go

```diff
@@ -176,7 +176,28 @@ func startServerWithOptions(cli *CliOptions) *apiserver.Server {
 		Version: "1.22", //dockerversion.Version,
 	}
 
-	tlsConfig := tlsconfig.ServerDefault
+	copyTLSConfig := func(c *tls.Config) *tls.Config {
```

Comment:
Does this need to be in an anonymous func?  I only see it used once immediately after declaring.